hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/BatchConsumer.java

```diff
@@ -112,18 +133,47 @@ public void consume(Runnable signalsInterrupt) {
                 metrics.markDiscarded();
                 trackers.get(subscription).logDiscarded(m, "too large");
             });
+        } catch (Exception e) {
```

Comment:
why add it here? I believe an error here is meant to be handled by `ConsumerProcess`